sonar-java-plugin/pom.xml

```diff
@@ -112,7 +101,7 @@
             <configuration>
               <rules>
                 <requireFilesSize>
-                  <maxsize>3600000</maxsize>
+                  <maxsize>6400000</maxsize>
                   <minsize>2000000</minsize>
```

Comment:
I would also modify `minsize`, in order to detect any suspicious change on that side.